### PR TITLE
Remove zuul host from SCS instance and unpdate endpoint URL

### PR DESCRIPTION
**Pull request by matofeder**
Zuul has been migrated to the k8s cluster, hence the old Zuul host (VM) monitoring could be removed.

Zuul UI URL has been changed to the new one.

### CODE REVIEW

File: values-observer-scs.yaml [hunk 1]
```diff
@@ -421,89 +421,6 @@ dnation-kubernetes-monitoring:
           - port: "9180"
             interval: 30s
             path: /metrics
-## Uncomment only when SCS Zuul VM has been bootstrap with node and cadvisor exporters
```
Comment (chess-knight) [inline values-observer-scs.yaml:424]: I see these sections also in values-observer, Should they also be deleted?

File: values-observer-scs.yaml [hunk 2]
```diff
@@ -421,89 +421,6 @@ dnation-kubernetes-monitoring:
...
-    - name: zuul
-      description: SCS Zuul VM
-      jobName: zuul-scs
-      host:
-        address: 213.131.230.42
```
Comment (chess-knight) [inline values-observer-scs.yaml:429]: I see some references to the `213.131.230.42` in zuul-zookeeper-dashboard. Is this dashboard still relevant?

File: values-observer-scs.yaml (hunk 1 above)
Comment (matofeder) [reply values-observer-scs.yaml:424]: IMO values-observer are generic, non SCS specific, so let the example of Zuul related values there

File: values-observer-scs.yaml (hunk 2 above)
Comment (matofeder) [reply values-observer-scs.yaml:429]: The zuul-zookeeper-dashboard is still relevant and could be used. 
Yep, the pre-defined selection of IP address should not be there, but it is not a major issue and could be removed later.

Comment (chess-knight) [reply values-observer-scs.yaml:429]: Ok, thank you

**Review by chess-knight — APPROVED** [review]

File: values-observer-scs.yaml (hunk 2 above)
Comment (matofeder) [reply values-observer-scs.yaml:429]: https://github.com/SovereignCloudStack/k8s-observability/pull/76

**Pull request merged**